app/talk/sticky-discussion-list.cjsx

```diff
@@ -1,8 +1,6 @@
+React = require 'react'
 talkClient = require '../api/talk'
 
-placeholder = document.createElement 'li'
```

Comment:
I've added a replacement for this, but sticky reordering doesn't seem to work for me at all in Firefox or Chrome on the main site anyway.